### PR TITLE
Added Feature type/Slideshow and view

### PR DESCRIPTION
**Pull request by KMDOFNI**
Added Feature type/Slideshow and view
Enabled block titles

### CODE REVIEW

**Review by soda736 — CHANGES_REQUESTED** [review]
Just a couple of changes but LG

**Comment by soda736** [comment]
You might also need to add Feature to workflow and scheduling /admin/config/workflow

File: project/config/pacni/config/core.entity_form_display.node.feature.default.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+uuid: c3e90331-7dc2-4b11-938d-475cb9f08e47
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.feature.body
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/config/pacni/config/core.entity_form_display.node.feature.default.yml]: Thanks I have removed this field

File: project/config/pacni/config/entityqueue.entity_queue.slideshow.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    target_bundles:
+      feature: feature
+    sort:
+      field: _none
+      direction: ASC
+    auto_create: false
+    auto_create_bundle: ''
+queue_settings:
+  min_size: 0
+  max_size: 1
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/config/pacni/config/entityqueue.entity_queue.slideshow.yml]: Updated this value to 5

**Comment by KMDOFNI** [comment]
> You might also need to add Feature to workflow and scheduling /admin/config/workflow

Feature is now added to Workflow and Scheduling

**Review by soda736 — APPROVED** [review]
LG

**Pull request merged**